tools/metams/metams_plot.xml

```diff
@@ -0,0 +1,245 @@
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    
+    <expand macro="requirements">
+        <requirement type="package" version="9.22">ghostscript</requirement>
+    </expand>
+
+    <required_files>
+        <include type="literal" path="metams_plot.r"/>
+    </required_files>
+
+    <command detect_errors="exit_code"><![CDATA[
+    @COMMAND_RSCRIPT@/metams_plot.r
+        
+        metaMS '$metaMS'
+
+        selecttic $selectedtic
+
+        selectbpc $selectedbpc
+
+        selecteic $selectedeic.EIC
+        
+        #if $selectedeic.EIC == "true"
+            unkn "c($selectedeic.unkn)"
+        #else
+            unkn "none"
+        #end if
+
+        @COMMAND_FILE_LOAD@
+
+    ]]></command>
+
+    <inputs>
+
+        <param name="metaMS" type="data" format="rdata" label="Rdata from new_metaMS_runGC" help="Rdata file containing a metaMS preprocess object" />
+        <param name="selectedtic" type="boolean" checked="false" falsevalue="false" truevalue="true" label="Do you want to process for TIC(s) ?" />
+        <param name="selectedbpc" type="boolean" checked="false" falsevalue="false" truevalue="true" label="Do you want to process for BPC(s) ?" />
+
+        <conditional name="selectedeic">
+            <param name="EIC" type="select" label="Do you want to process for EIC(s) ?">
+                <option value="false">NO</option>
+                <option value="true">YES</option>
+            </param>
+            <when value="true">
+                <param name="unkn" type="text" value="1:5" label="EIC_Unknown" help="vector of peaks number to be plotted, for example 1:5 (mean 1 to 5) or 1,4,12  (means 1 4 and 12). For all EIC use 0" /> 
+            </when>
+            <when value="false"/>
+        </conditional>
+
+        <expand macro="input_file_load"/>
+
+    </inputs>
+
+    <outputs>
+    
+        <!-- Only when user selected TIC option -->
+        <data name="ticsRawPdf"   format="pdf" from_work_dir="TICs_raw.pdf" label="TICs_raw.pdf">
+            <filter>selectedtic is True</filter>
+        </data>
+        <!-- Only when user selected BPC option -->
+        <data name="bpcsRawPdf"   format="pdf" from_work_dir="BPCs_raw.pdf" label="BPCs_raw.pdf">
+            <filter>selectedbpc is True</filter>
+        </data>
+        <!-- Only when user selected EIC option -->
+        <data name="unknownPdf"   format="pdf" from_work_dir="GCMS_EIC.pdf" label="GCMS_EIC.pdf">
+            <filter>selectedeic['EIC'] is True</filter>
+        </data>
+
+    </outputs>
+
+    <tests>
+
+        <!-- Test TIC -->
+        <test expect_num_outputs="1">
+            <param name="metaMS" value="runGC_test.RData"  ftype="rdata" />
+            <expand macro="test_file_load_alg_symlink" />
+            <param name="selectedtic" value="true"/>
+            <output name="ticsRawPdf" value="TICs_raw.pdf" ftype="pdf" compare="sim_size" delta="600"/>
```

Comment:
Instead of saving the file for comparison and having a `sim_size` comparison we could use https://docs.galaxyproject.org/en/master/dev/schema.html#has-size with equivalent results